packages/vuetify/src/components/VBtn/__tests__/VBtn.spec.ts

```diff
@@ -96,6 +96,17 @@ describe('VBtn.ts', () => {
     expect(wrapper.html()).toMatchSnapshot()
   })
 
+  it('should should render disabled on an <a> tag when using href prop', () => {
```

Comment:
double should

Reply:
Because I believe it so strongly! Hahah, thanks, will fix.